homework3.php

```diff
@@ -0,0 +1,21 @@
+<?php
+//homework3
+function checkPrimeNumber($n) {
+    if($n == 1){
+      echo"1 isn't Prime Number";
+    }else{
+      $i=2;
+      while($i<=$n){
+        if($n % $i == 0 && $n != $i){
+          echo "Not Prime Number";
+          break;
+        }else if($n % $i != 0){
+          ++$i;
+        }else{
+            echo $n." is Prim Number";
+            ++$i;
+          }
+        }
+      }    
```

Comment:
Because `checkPrimeNumber`  is only for checking a number is prime or not. So that it should return `true` or `false` instead of output result on the screen.